pkg/kubelet/cm/cgroup_manager_linux.go

```diff
@@ -449,12 +449,16 @@ func (m *cgroupManagerImpl) Pids(name CgroupName) []int {
 
 		// WalkFunc which is called for each file and directory in the pod cgroup dir
 		visitor := func(path string, info os.FileInfo, err error) error {
+			if err != nil {
+				glog.V(5).Infof("cgroup manager encountered error scanning cgroup path %q", path)
+				return filepath.SkipDir
```

Comment:
@vishh would you say it is equally safe as skipping 10 lines later if there's an error reading cgroup data?